cli/ts/deployPollWithSigner.ts

```diff
@@ -188,8 +190,15 @@ const deployPollWithSigner = async (args: any) => {
     // Deploy a PollProcessorAndTallyer contract
     const verifierContract = await deployVerifier(true)
     console.log('Verifier:', verifierContract.address)
-    const pptContract = await deployPpt(verifierContract.address, true)
-    await pptContract.deployTransaction.wait()
+
+    const mpContract = await deployMessageProcessor(verifierContract.address, contractAddrs['PoseidonT3'],contractAddrs['PoseidonT4'],contractAddrs['PoseidonT5'],contractAddrs['PoseidonT6'])
```

Comment:
I'm confused - seems like we're deploying additional contracts here? Why if so? From the desc I thought this PR was just focused on simplifying the workflows, not changing any deploy functionality.

Reply:
this PR aims to fix the various errors in the CI tests, one of those was due to this line missing in one of the latest cli utilities that were introduced.